CircleciScripts/setup_private_specs.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
```

Comment:
Not exactly sure what this script is doing or when it is suppose to be run.  Would it be possible to get a high level description at the top of the file of it's purpose?

Reply:
Noted in description: The setup_private_specs.sh script also makes it possible to lint the pods before the release. It modifies the specs in the private repo so dependencies resolve correctly, but the base file itself is linted directly from the local git working directory. I have validated that all of Amplify.podspec, AmplifyPlugins.podspec, AWSPredictionsPlugin.podspec, and CoreMLPredictionsPlugin.podspec pass with pod lib lint --sources=amplify-ios-local-specs,trunk <podspec>.

This will be part of official documentation when I have time to put it all together, but that will be a post-RC task

Reply:
thanks!